conda-forge.yml

```diff
@@ -1,5 +1,7 @@
 appveyor:
   secure: {BINSTAR_TOKEN: tumuXLL8PU75WMnRDemRy02ruEq2RpNxeK3dz0MjFssnosPm2v4EFjfNB4PTotA1}
+provider:
+  linux_ppc64le: default
```

Comment:
Can you add `linux_aarch64` as well?

Reply:
I added that. Is there a reason that not adding `linux_aarch64` could be an issue? Or is it just that conda-forge would prefer to have both available?